ui/app/routes/vault/cluster/secrets/backend/configuration/index.js

```diff
@@ -149,6 +129,49 @@ export default class SecretsBackendConfigurationRoute extends Route {
     }
   }
 
+  async fetchGcpConfig(id) {
+    try {
+      const gcpModel = await this.store.queryRecord('gcp/config', { backend: id });
```

Comment:
if the request for `gcpModel` fails wouldn't the value still be truthy? It'd just be an adapter error object, right?